internal/syncd/integrations/ecr/reconciler.go

```diff
@@ -2,27 +2,55 @@ package ecr
 
 import (
 	"context"
+	"fmt"
+	"path/filepath"
 
 	"ship-it/internal"
 
+	"ship-it/pkg/apis/k8s.wattpad.com/v1alpha1"
+
 	"github.com/google/go-github/github"
+	"gopkg.in/yaml.v2"
 )
 
 type RepositoriesService interface {
-	GetContents(ctx context.Context, org, repo, path string, opts *github.RepositoryContentGetOptions) (*github.RepositoryContent, []*github.RepositoryContent, *github.Response, error)
-	UpdateFile(ctx context.Context, owner, repo, path string, opt *github.RepositoryContentFileOptions) (*github.RepositoryContentResponse, *github.Response, error)
+	UpdateFile(msg string, branch string, path string, fileContent []byte) (*github.RepositoryContentResponse, error)
+	GetFile(branch string, path string) ([]byte, error)
 }
 
 type ImageReconciler struct {
-	// TODO
+	Org          string
+	ResourcePath string
```

Comment:
This approach will only work for a single resource on a single branch. We need to be able to make changes to multiple HelmRelease files (probably for the same branch, `master`), so the path to the CR will probably need to be configured from env.

The branch can be fixed, but it should be configurable from env and we can default it to `master`

Reply:
I have added some ECR related environment variables to the `syncd` config struct. The release branch is an environment variable with a default value of `master`